src/layers/trip-layer/trip-layer.js

```diff
@@ -215,7 +200,8 @@ export default class TripLayer extends Layer {
 
     return {
       data: geojsonData,
-      getFeature,
+      getPath: d => d.geometry.coordinates.map(coord => coord.slice(0, 2)),
```

Comment:
You can probably pass `d => d.geometry.coordinates`, no need to map it. the 4th element is not used in deck.gl. Let me know if this doesn't work